components/globals/Loader.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { css } from "@emotion/react";
+import Loader from "react-spinners/PacmanLoader";
```

Comment:
As much as I love it, I feel like the PacmanLoader might not be the _best_ choice.. 🤔

Reply:
lol I know... It's more of a fun placeholder until design provides their input.  I can also swap it out for something more 'standard'.